templates/terraform/monster-infrastructure/clinvar.tf

```diff
@@ -1,22 +1,16 @@
-provider google-beta {
-  project = "broad-dsp-monster-clingen-dev"
-  region = "us-central1"
-  alias = "clinvar"
-}
-
 module clinvar {
   source = "/templates/processing-project"
   providers = {
-    google.target = google-beta.clinvar,
-    vault.target = vault.command-center
+    google.target = google.clinvar,
+    vault.target = vault.target
   }
 
-  project_name = "broad-dsp-monster-clingen-dev"
+  project_name = "broad-dsp-monster-clingen-${var.env}"
   is_production = false
```

Comment:
This will need to toggle based on `env`. Alt: the mega-module should take this flag and use it to compute a `local` for the env string.

Reply:
AH good catch